### DISCUSSION TITLE
image size

### DISCUSSION THREAD

**Discussion by SylvainArd** [facebookresearch/detectron2#4690] [General]
Hello,
I want to put 3500 X 2500 pixels images in the input of neural network of detectron2. How to do this please ?
Thank you
Best regards

**Comment by SylvainArd** [comment]
well 2500x3500

**Comment by SylvainArd** [comment]
up please

**Comment by SylvainArd** [comment]
up please

**Comment by SylvainArd** [comment]
up please

**Comment by pkhateri** [comment]
Follow the steps in this [notebook](https://colab.research.google.com/drive/16jcaJoc6bCFAQ96jDe2HwtXj7BMD_-m5#scrollTo=PIbAM2pv-urF), and modify this function `get_balloon_dicts(img_dir)` for your own data.
